workshops/vue/minis/mini6.md

```diff
@@ -517,10 +527,13 @@ Edit **Quiz.vue**'s template to show a dynamic image at the top, right under the
 ```html
 <!-- Quiz.vue -->
 <template>
-    <div>
-    <img :src="image" alt>
+  <div>
+    <img :src="image" alt />
     ...
+  </div></template
+>
```

Comment:
remove the closing `</div>` and `</template>` tags here that the linter added, as this is not a complete code block for teaching purposes.